docs/src/pages/start/pick-quasar-flavour.md

```diff
@@ -21,22 +19,31 @@ Here's a quick comparison:
-| Build Browser Extensions                                                            | -       | **Yes**    | Yes*                          |
+| Build SSR (+ optional PWA client takeover)                                                 | -       | **Yes**    | Yes(*)                          |
+| Build Mobile Apps via Cordova or Capacitor                                                 | **Yes** | **Yes**    | Yes(*)                          |
+| Develop Mobile Apps with HMR directly on your phone.                                       | -       | **Yes**    | Yes(*) |
+| Build Desktop Apps via Electron                                                            | -       | **Yes**    | Yes(*)                             |
+| Build Browser Extensions                                                                   | -       | **Yes**    | Yes(*)                          |
 | Quasar **App Extensions**                                                                  | -       | **Yes**    | - |
-| App icons & splash screens easy management through [Icon Genie](/icongenie/introduction) | - | **Yes** | - |
+| Easy management of App icons & splash screens via [Icon Genie CLI](/icongenie/introduction)    | - | **Yes** | - |
 | Dynamic RTL support for Quasar components                                                  | **Yes** | **Yes**    | **Yes**                       |
 | Generating your own website/app RTL equivalent CSS rules automatically by Quasar           | -       | **Yes**    | **Yes**                       |
 | **Ensure everything "simply works" out of the box**, using latest and greatest Quasar specs.   | -       | **Yes**    | -                             |
 | **Tight integration between build modes**, taking full advantage of all Quasar's capabilities. | -       | **Yes**    | -                             |
-| Develop mobile apps with HMR directly on your phone.                                       | -       | **Yes**    | Yes*                          |
-| One codebase to rule SPA, PWA, SSR, Mobile Apps, Electron Apps                             | -       | **Yes**    | Yes*                          |
+| One codebase to create SPA, PWA, SSR, Mobile Apps, Electron Apps and Browser Extensions                             | -       | **Yes**    | Yes(*)                      |
 | Tree Shaking                                                                               | -       | **Yes**    | **Yes**                       |
 | SFC (Single File Component - for Vue) support                                              | -       | **Yes**    | **Yes**                       |
 | Advanced configuration through dynamic quasar.conf.js                                      | -       | **Yes**    | -                             |
 | Unit & end to end testing                                                                  | -       | **Yes**    | **Yes**                       |
```

Comment:
This should probably be renamed to "Unit & end to end testing integration" or "Unit & end to end testing support", as Quasar isn't automatically unit or e2e tested per se

Reply:
I think it's fairly clear the project target is meant, not the core software. :) But, I'll make the change, so it's definitely not mistaken.